src/components/MarkdownProvider/components/Configuration.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { UnorderedList, Span } from '@zendeskgarden/react-typography';
+import { Anchor } from '@zendeskgarden/react-buttons';
+import { getColor } from '@zendeskgarden/react-theming';
+import { ComponentDoc } from 'react-docgen-typescript';
+
+export interface IPackage {
+  version: string;
+  name: string;
+  peerDependencies: string[];
+  packageName: string;
+}
+
+const StyledUnorderedList = styled(UnorderedList)`
+  margin: 0 0 ${p => p.theme.space.xxl};
+  border-bottom: ${p => `${p.theme.borders.sm} ${getColor('grey', 300, p.theme)}`};
+`;
+
+const StyledListItem = styled(UnorderedList.Item)`
+  list-style: none;
+
+  & > div {
+    display: flex;
+    margin: ${p => p.theme.space.sm} 0;
+    border-top: ${p => `${p.theme.borders.sm} ${getColor('grey', 300, p.theme)}`};
+    padding: ${p => p.theme.space.sm} 0 0;
+    font-family: ${p => p.theme.fonts.mono};
+  }
+`;
+
+const StyledListItemLabel = styled.label`
+  min-width: ${p => p.theme.space.base * 20}px;
+  font-family: ${p => p.theme.fonts.system};
+  font-weight: ${p => p.theme.fontWeights.semibold};
+`;
+
+const StyledSpan = styled(Span)`
+  margin-right: ${p => p.theme.space.sm};
+`;
+
+export const Configuration: React.FC<{ reactPackage: IPackage; propSheets: ComponentDoc[] }> = ({
+  reactPackage,
+  propSheets
+}) => {
+  return (
+    <StyledUnorderedList>
+      <StyledListItem>
+        <StyledListItemLabel>Name</StyledListItemLabel>
+        <StyledSpan>{reactPackage.version}</StyledSpan>
+        <StyledSpan>•</StyledSpan>
+        <StyledSpan>
+          <Anchor
+            href={`https://github.com/zendeskgarden/react-components/tree/master/packages/${reactPackage.packageName}`}
+            target="_blank"
+          >
+            View source
+          </Anchor>
+        </StyledSpan>
+        <StyledSpan>•</StyledSpan>
+        <StyledSpan>
+          <Anchor
+            href={`https://www.npmjs.com/package/@zendeskgarden/react-${reactPackage.packageName}`}
+            target="_blank"
+          >
+            View on NPM
+          </Anchor>
+        </StyledSpan>
+      </StyledListItem>
+      <StyledListItem>
+        <StyledListItemLabel>Install</StyledListItemLabel>
+        npm install {reactPackage.name}
+      </StyledListItem>
+      <StyledListItem>
+        <StyledListItemLabel>Deps</StyledListItemLabel>
+        npm install {reactPackage.peerDependencies.join(' ')}
+      </StyledListItem>
+      <StyledListItem>
+        <StyledListItemLabel>Import</StyledListItemLabel>
+        import{' '}
+        {`{ ${propSheets && propSheets.map(propSheet => propSheet.displayName).join(', ')} }`} from
+        &apos;
+        {reactPackage.name}
+        &apos;
```

Comment:
These imports are normally shown at the `propSheet` level, not at the package configuration level. What is the intention with this line?

Reply:
To show the import command next to the import label in the configuration section.

Reply:
For some pages this will include 10+ components. This feels like duplicated info from the individual prop sheet area.

With us splitting packages between documentation pages this also only shows a subset of the available imports.

Reply:
The import statements were pulled from above each individual API sheet and put in the Configuration section as a collection to see what's available as a group so there is one reference point.

[Here is the tentative design spec](https://share.getcloudapp.com/12uNeGeE) I was working against but @allisonacs may have additional revisions.

Reply:
@austin94 is your suggestion to remove this list item entirely? That makes sense to me, but I just wanted to confirm if I understood your comment.

Reply:
That feels like a good path to me. Imports are used for including a specific component. I don't see anyone using this value directly.

Reply:
I'm unsure what to do here. I think what we're proposing is a change from the design spec I was working with which is fine, but I think @allisonacs is taking that on. I will defer to direction on this. Please let me know if I should move the import statements back to the top of each API table component to match the current Garden website design or continue with the new design spec.